packages/react/src/components/Switch/Switch.js

```diff
@@ -32,9 +43,17 @@ const Switch = props => {
   const handleKeyDown = e => {
     const key = e.key || e.which;
 
-    if (key === 'Enter' || key === 13 || key === ' ' || key === 32) {
+    if (
+      match(key, keys.SPACE || keys.ENTER || keyCodes.SPACE || keyCodes.ENTER)
+    ) {
       onKeyDown({ index, name, text });
     }
+    if (match(key, keyCodes.RIGHT)) {
```

Comment:
these two blocks can be combined right?

```suggestion
    if (match(key, keyCodes.RIGHT) || match(key, keyCodes.LEFT) {
```